plugins/processors/awsappsignals/factory.go

```diff
@@ -33,9 +35,8 @@ func NewFactory() processor.Factory {
 }
 
 func createDefaultConfig() component.Config {
-	return &Config{
-		// TODO: change default config when other resolvers are supported
-		Resolvers: []string{"eks"},
+	return &appsignalsconfig.Config{
+		Resolvers: []appsignalsconfig.Resolver{},
```

Comment:
is there a use case when we have a default resolver? What happens then?

Reply:
I would never expect the default resolver to be initied for real use case. In case it happens, it will fail the [config validation](https://github.com/aws/amazon-cloudwatch-agent/pull/980/files#diff-0c90c470e167f05f8e26955d6fa96fbef610a649259dc966668e3d6ccf11331bR29) with `unknown resolver` error.